frontend/src/selectors/mineSelectors.js

```diff
@@ -36,4 +36,11 @@ export const getCurrentPermitteeIds = createSelector(
     unique = [...new Set(permitteeIds)];
     return unique;
   }
-);
\ No newline at end of file
+);
+
+export const getMineRegionHash = createSelector([getMineRegionOptions], (regions) => {
+  return regions.reduce(
```

Comment:
If we care, Airbnb linting rules would not like this immediate `return` in an arrow function